features/bootstrap/ApiContext.php

```diff
@@ -210,4 +211,26 @@ public function iShouldReceiveAnCaptureResponse()
         }
     }
 
+    /**
+     * @When I send the following refund modification
+     */
+    public function iSendTheFollowingRefundModification(TableNode $table)
+    {
+        $this->response = $this->application->refundPayment($table->getRowsHash());
+    }
+
+    /**
+     * @Then I should receive an refund response
+     */
+    public function iShouldReceiveAnRefundResponse()
+    {
+        if (!$this->response instanceof RefundResponse) {
```

Comment:
Can use `Assert::isInstanceOf($this->response, RefundResponse::class);` instead.